abouts.html

```diff
@@ -0,0 +1,145 @@
+             <li><a href="#" class="link-nav">Program</a></li>
+             <li><a href="#" class="link-nav">Sponsor</a></li>
+             <li><a href="#" class="link-nav">Join</a></li>
+          </ul>
+        </div>
+
+        <!-- end of mobile menu dynamic -->
+  
+      <div class="desktop-menus">
+          <a href="#" class="le-logo"><img src="./images/lemorsels-logo.PNG" alt="#" class="le-logo2"></a> 
+  
+           <div class="desktop-nav-lnks">
+              <ul class="menu-ul-lnks">
+                  <li><a href="./abouts.html" class="menu-links">About</a></li>
+                  <li><a href="#" class="menu-links">Program</a></li>
+                  <li><a href="#" class="menu-links">Join</a></li>
+                  <li><a href="#" class="menu-links">Sponsor</a></li>
+                  <li><a href="#" class="menu-links">News</a></li>
+              </ul>
+              <button class="menu-button" type="button">AS Campaign</button>
+            </div>
+          </div>
+       </nav>
+    </header>
+        
+           <section class="headline-container">
+  
+              <div class="Le-title">
+                 <p class="catch-phrase">"Hello, organic food lovers!"</p>
+                 <h1 class="Le-Slogan">LeMorsels, IT'S NATURAL AND ORGANIC AS IT SHOULD BE!</h1>
+              </div>
+
+            <div class="below-slogan">
+              <p class="Lemorse-quote">
+                Here in LeMorsels we are a research organization that wants more food products to be free of artificial food additives like artificial sweeteners, preservatives, coloring, flavoring, and monosodium glutamate (MSG).
+                Some of our speakers surpervise if farmers are using organically grown crops tend to use natural fertilizers like manure to improve plant growth. The good of using organic grown crops is that animals raised organically don't need to much antibiotics or hormones.
+                The most commonly purchased organic foods are fruits, vegetables, grains, dairy products and meat.</p>
+                
+                <p class="lemorse-mssg"> Please reach out to us, through our Email below for any doubt or further questions
+                  about LeMorsels Organic research lab.</p>
+
+                <a href="#" class="cntc-email">le_organicrslb@levnz.org</a>
+   
+            </div>
+          </section>
+         </div>
+
+         <section class="memorable-logo">
+          <h2 class="memorable-logo-tt">LeMorsels Organic Research Lab</h2>
+          <div class="underline"></div>
+          <p class="competition-logo">The logo of LM Organic Research was , which was decided through a once in a time Logo competition which was held from 8 of June to 7 of July. 
+            The winning designer received a special book from LeMorsels and the designer let us optimzed for use on the website and promotional materials.</p>
+      
+          <img src="./images/lemorsels-logo.PNG" alt="#" class="big-logo">  
```

Comment:
- Kindly fill the `alt` attribute with values for those without visual accessibility. It's a project requirement.